downloads/index.md

```diff
@@ -95,6 +95,41 @@ the Juno executable or app.
 
 ---
 
+# Release Candidates (v0.4.0-rc1)
+
+We are currently testing release candidates for the next major version of Julia.  Please see
```

Comment:
I'm looking for the reference, but I remember @StefanKarpinski saying somewhere that before 1.0, versions such as `0.4` and `0.3` were considered "major" versions.  Since there does seem to be some confusion regarding this, I'll just remove "major" and say "the next version of Julia".